tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regexp.extract.full.txt

```diff
@@ -0,0 +1,212 @@
+<results>
+  <test name='REGEXP_EXTRACT([str0], &apos;(E.{3})&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(2625345313)(0)]</column>
+      </schema>
+      <tuple>
+        <value>%null%</value>
+      </tuple>
+      <tuple>
+        <value>&quot;E SU&quot;</value>
+      </tuple>
+      <tuple>
+        <value>&quot;ECHN&quot;</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='REGEXP_EXTRACT([key], &quot;(key)&quot;) //&quot;key&quot;'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(896336668)(0)]</column>
+      </schema>
+      <tuple>
+        <value>&quot;key&quot;</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='REGEXP_EXTRACT([str0], &quot;(z)&quot;) //&quot;Null&quot;'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(3326887800)(0)]</column>
+      </schema>
+      <tuple>
+        <value>%null%</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='REGEXP_EXTRACT(&quot;toyota&quot;, &quot;(o)&quot;) //&quot;o&quot;'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(1576583407)(0)]</column>
+      </schema>
+      <tuple>
+        <value>&quot;o&quot;</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='REGEXP_EXTRACT([num0], &quot;(0{2})&quot;) //&quot;00&quot;'>
```

Comment:
This expected isn't right, it's throwing an error. Need to update the test file to point to something other than a col. I'll leave a comment below for that.